.github/workflows/cache.yaml

```diff
@@ -0,0 +1,32 @@
+name: cache
+on:
+  workflow_call:
+  workflow_dispatch:
+concurrency:
+  group: cache-${{ github.ref }}
+  cancel-in-progress: true
+permissions:
+  actions: write
+jobs:
+  clear-cache:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Clear cache
+        uses: actions/github-script@v6
+        with:
+          script: |
+            console.log("Preparing to clear cache...")
+            const caches = await github.rest.actions.getActionsCacheList({
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+            })
+            for (const cache of caches.data.actions_caches) {
+              console.log("Clearing cache_id", cache.id)
+              await github.rest.actions.deleteActionsCacheById({
+                owner: context.repo.owner,
+                repo: context.repo.repo,
+                cache_id: cache.id,
+              })
+              console.log("Cleared cache_id", cache.id)
+            }
+            console.log("Cleared cache.")
```

Comment:
This looks good, but how can be we confident that this works as expected before merging?

Reply:
Unfortunately, workflows can’t be manually triggered until they’re in the main branch. Perhaps a new testing draft pr based on this one where I commit a push trigger to a workflow…

Reply:
That's what I was thinking and suggested to @AntBush  for #592

Reply:
Runs successfully and deletes all cache layers
https://github.com/bcgov/CONN-CCBC-portal/actions/runs/3062028759/jobs/4942521697

Reply:
beautiful. We should note this as a way to test GH workflows